### DISCUSSION TITLE
Copilot does not give the full suggestion

### DISCUSSION THREAD

**Discussion by hilalmr** [community/community#11002] [Copilot Conversations]
when you type something for the github copilot to do it does not do it until we start something

**Comment by Chiroyce1** [comment]
As the name suggests, it's meant to guide you and not write full code on it's own. Try writing a few comments in your code to tell Copilot what to do.

**Comment by davecheney** [comment]
Can you show me what you did? A screenshot would be great.

Also, can you please paste the relevant parts of View -> Output; select _GitHub Copilot_ from the pulldown.
